core/tests/unit/kmx/meson.build

```diff
@@ -89,29 +89,22 @@ tests = [
   '048 - modifier keys keep context'
 ]
 
+node = find_program('node', required: true)
+
+kmc_root = meson.current_source_dir() / '../../../../developer/src/kmc/build/src/kmc.js'
+kmc_cmd = [node, kmc_root]
+
 if build_machine.system() == 'windows'
-  kmcomp = find_program(join_paths(meson.current_source_dir(),'..','..','..','..','windows','bin','developer','kmcomp.exe'), 'kmcomp.exe', required: false)
-  kmcomp_cmd = [kmcomp]
   copy_cmd = [find_program('cmd.exe', required: true), '/c', 'copy']
   cat_cmd = [find_program('cmd.exe', required: true), '/c', 'type']
 else
-  kmcomp = find_program('kmcomp', required: false)
-  kmcomp_cmd = [kmcomp]
-  if not kmcomp.found()
-    wine = find_program('wine', required: false)
-    kmcomp = find_program('kmcomp.exe', required: false)
-    kmcomp_cmd = [wine, kmcomp]
-    if not wine.found()
-      kmcomp = disabler()
-    endif
-  endif
   copy_cmd = [find_program('cp', required: true)]
   cat_cmd = [find_program('cat', required: true)]
 endif
 
 if cpp_compiler.get_id() == 'emscripten'
```

Comment:
Is it worth assigning this to a boolean? I see check happens 5x in the file.

Reply:
I think I will be doing some more cleanup on the meson.build files in coming weeks, so will keep this in consideration